styles/styles.css

```diff
@@ -123,19 +139,19 @@ nav {
 .landing-pg-text {
   text-align: center;
   max-width: 70rem;
-  font-size: 2rem;
-  font-weight: 500;
+  font-size: 3.5rem;
+  font-weight: 600;
   margin: 4rem;
 }
 
 .landing-pg-btn {
   text-decoration: none;
-  font-size: 1.7rem;
+  font-size: 2rem;
   font-weight: 500;
   color: var(--black-color);
   padding: 1rem 2rem;
   margin: 1rem;
-  border-radius: 2rem;
+  border-radius: 3rem;
   box-shadow: 0px 1px 8px rgba(0, 0, 0, 0.3);
 }
```

Comment:
You can use colour variables here as well.

Reply:
yes, made the variable for box shadow after hover